src/components/customerPortal/PortalInvoicesList.tsx

```diff
@@ -68,10 +76,50 @@ gql`
     }
   }
 
+  query getCustomerPortalInvoicesCollection($expireCache: Boolean) {
+    customerPortalInvoiceCollections(expireCache: $expireCache) {
```

Comment:
Broader topic, I'm wondering if this endpoint could be "melted" into the `customerPortalInvoices` used above.
That would require some BE rework and probably be skipped, but here is my path
- I feel we query the same resource in both endpoints
- `expireCache` seems not to be used here
- `amountCents` is probably the total amount cents of all invoices added. Wondering if that could be added to the metadata
- `invoicesCount` is probably the `customerPortalInvoices`' `metadata.totalCount`, or as we han have filters in `customerPortalInvoices` it should be another metadata
- the `currency` got there should be the customer's one, fetched bellow in `getCustomerPortalUserCurrency` query
Then this query could be replaced.

That's form my understanding I may miss something tho, let me know!

Reply:
We decided to tackle this later